### ISSUE TITLE
Using Systemd's "sysctl stop/start/reload lizardfs-chunkserver" after "mfschunkserver stop/start/reload" does not work

### ISSUE THREAD

**Issue by Asher256** [lizardfs/lizardfs#451]
When I start mfschunkserver with the command "mfschunkserver start" I'm no longer able to use "systemctl start/stop/status mfschunkserver". 

My operating system: Debian Jessie

To reproduce the bug, start mfschunkserver with this command:
```
mfschunkserver start
```

And check the status with systemd (or start/stop/reload mfschunkserver with systemctl):
```
systemctl status mfschunkserver
```

Even though mfschunkserver is started and running, Systemd is unable to check the status:
```
shp-p-lfs004: root /var/lib systemctl status lizardfs-chunkserver
● lizardfs-chunkserver.service - LizardFS chunkserver daemon
   Loaded: loaded (/lib/systemd/system/lizardfs-chunkserver.service; disabled)
   Active: failed (Result: exit-code) since Thu 2016-08-18 18:48:24 UTC; 2min 8s ago
     Docs: man:mfschunkserver
  Process: 25516 ExecStart=/usr/sbin/mfschunkserver -d start (code=exited, status=2)
 Main PID: 25516 (code=exited, status=2)

Aug 18 18:48:24 shp-p-lfs004 mfschunkserver[25516]: configuration file /etc/lizardfs/mfschunkserver.cfg loaded
Aug 18 18:48:24 shp-p-lfs004 mfschunkserver[25516]: set gid to 116
Aug 18 18:48:24 shp-p-lfs004 systemd[1]: lizardfs-chunkserver.service: main process exited, code=exited, status=2/INVALIDARGUMENT
Aug 18 18:48:24 shp-p-lfs004 systemd[1]: Failed to start LizardFS chunkserver daemon.
Aug 18 18:48:24 shp-p-lfs004 systemd[1]: Unit lizardfs-chunkserver.service entered failed state.
Aug 18 18:48:24 shp-p-lfs004 mfschunkserver[25516]: [ OK ] changed working directory to: /var/lib/lizardfs
Aug 18 18:48:24 shp-p-lfs004 mfschunkserver[25516]: [FAIL] can't start: lockfile /var/lib/lizardfs/.mfschunkserver.lock is already locked by another process
Aug 18 18:49:06 shp-p-lfs004 systemd[1]: Unit lizardfs-chunkserver.service cannot be reloaded because it is inactive.
Aug 18 18:49:13 shp-p-lfs004 systemd[1]: Unit lizardfs-chunkserver.service cannot be reloaded because it is inactive.
Aug 18 18:50:23 shp-p-lfs004 systemd[1]: Unit lizardfs-chunkserver.service cannot be reloaded because it is inactive.
```

It is also impossible to stop the process with systemd:
```
systemctl stop lizardfs-chunkserver
```

(I need to use "mfschunkserver stop" instead)

I had the same issue with mfsmaster and mfsmetalogger. 

My suggestion: systemd should use the same functions as mfschunkserver start/stop/reload (or the opposite) to manage the daemons. This would be interesting for automated systems like Ansible or Puppet.

**Comment by psarna** [comment]
It is not a bug. `mfschunkserver` is an executable binary, which can be used to start/stop/reload/restart a chunkserver. It has little to do with systemd, /etc/init.d and other service managers.
What you did here was basically:
1. Run a command manually
`mfschunkserver start`
2. Check if external service manager that happens to use the same binary acknowledged that "lizardfs-chunkserver" service is running
`systemctl status lizardfs-chunkserver`
which is doomed to fail.

It is a perfectly correct behaviour that `systemctl stop lizardfs-chunkserver` did not stop your chunkserver. It tried to stop a `lizardfs-chunkserver` service, which was never launched in the first place, so it failed.

Systemd **does** use the same commands - see https://github.com/lizardfs/lizardfs/blob/master/rpm/service-files/lizardfs-chunkserver.service . It doesn't mean, however, that running `mfschunkserver` binary manually should cause systemd to acknowledge that a service was started - because it wasn't. Starting a service is well defined in systemd and can be performed with `systemctl start lizardfs-chunkserver`.

tl;dr: `mfschunkserver` is a binary, `lizardfs-chunkserver` is a service that manages a chunkserver by using `mfschunkserver` binary internally. Executing `mfschunkserver` manually does not, and will not, make systemd acknowledge that a service was started.
